src/App.jsx

```diff
@@ -1,5 +1,54 @@
+/** @jsxImportSource @emotion/react */
+import { css } from '@emotion/react';
+import Header from './Components/Header';
```

Comment:
이 경우 디렉토리 이름이 소문자인 것이 좋겠네요.

```suggestion
import Header from './components/Header';
```